src/Public/Logging.ps1

```diff
@@ -142,7 +179,51 @@ function New-PodeLoggingMethod {
 
         [Parameter(ParameterSetName = 'Custom')]
         [object[]]
-        $ArgumentList
+        $ArgumentList,
+
+        [Parameter(Mandatory = $true, ParameterSetName = 'Syslog')]
+        [switch]
+        $Syslog,
+
+        [Parameter(Mandatory = $true, ParameterSetName = 'Syslog')]
+        [string]
+        $Server,
+
+        [Parameter( ParameterSetName = 'Syslog')]
+        [Int16]
+        $Port = 514,
+
+        [Parameter( ParameterSetName = 'Syslog')]
+        [ValidateSet('UDP', 'TCP', 'TCPwithTLS', 'Splunk', 'LogInsight')]
```

Comment:
`TCPwithTLS` seems to be just `TLS` in most places, I'm guessing this is supposed to be just `TLS` a well?

Reply:
yes I changed everywhere but I forgot here.
this is still a work in progress :)